core/pom.xml

```diff
@@ -133,10 +133,8 @@
             <artifactId>commons-email</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <version>RELEASE</version>
-			<scope>test</scope>
+            <groupId>org.mockito</groupId>
```

Comment:
Fixed some tabs vs spaces here.  The file is pretty dirty with tabs right now, if someone wants to make a sweeping update.